lib/Raven/Configuration.php

```diff
@@ -824,7 +824,8 @@ private function configureOptions(OptionsResolver $resolver)
             return true;
         });
 
-        $resolver->setNormalizer('server', function (Options $options, $value) {
+        $resolver->setNormalizer('server', function (/* @noinspection PhpUnusedParameterInspection */
```

Comment:
I know @nokitakaze you like to have your IDE inspections to be all green, but we can't fill the whole project of annotations just for this... Also, it makes the code less readable than before to have this comment there. At least would it be possible to move it inside the function and get the same result?

Reply:
>we can't fill the whole project of annotations just for this

Why? I think I am not the only one who uses Idea

> would it be possible

I'm afraid no

Reply:
The point is to not add tool-specific stuff to the codebase. I'm :-1: too.

Reply:
>Why? I think I am not the only one who uses Idea

I use PHPStorm too, and I think a lot of other people too, but this does not means that we absolutely need to have all inspections to pass just because we don't use a parameter that unfortunately must be in the signature because of it's position